spec/instant.html

```diff
@@ -596,8 +596,12 @@ <h1>
         <dd>It computes the difference between two exact times _ns1_ and _ns2_ expressed in nanoseconds since the epoch, and rounds the result according to the parameters _roundingIncrement_, _smallestUnit_, _largestUnit_, and _roundingMode_.</dd>
       </dl>
       <emu-alg>
-        1. Assert: The following step cannot fail due to overflow in the Number domain because abs(ℝ(_ns2_) - ℝ(_ns1_)) &le; 2 &times; nsMaxInstant.
-        1. Let _roundResult_ be ! RoundDuration(0, 0, 0, 0, 0, 0, 0, 0, 0, ℝ(_ns2_) - ℝ(_ns1_), _roundingIncrement_, _smallestUnit_, _roundingMode_).[[DurationRecord]].
+        1. Let _difference_ be ℝ(_ns2_) - ℝ(_ns1_).
+        1. Let _nanoseconds_ be _difference_ modulo 1000.
+        1. Let _microseconds_ be (_difference_ / 1000) modulo 1000.
+        1. Let _milliseconds_ be (_difference_ / 10<sup>6</sup>) modulo 1000.
+        1. Let _seconds_ be _difference_ / 10<sup>9</sup>.
```

Comment:
I believe these should use [remainder](https://tc39.es/proposal-temporal/#eqn-remainder) instead of modulo so that they have the same sign as _difference_. E.g., for a _difference_ of -1999999999 we want -1, -999, -999, -999.

We also need to perform truncate() on the results of the divisions so that we get integers, because RoundDuration requires integers.